### ISSUE TITLE
Randomize changes hex pattern order globally

### ISSUE THREAD

**Issue by JoshuaStorm** [spiricom/MantaMate#94]
The "randomize all" button seems to change the pattern globally, which I don't think we intend to?

**Comment by spiricom** [comment]
as in it changes it for both sequences?

**Comment by JoshuaStorm** [comment]
Oh, hmm I didn't actually check that...
I'll explain this in person sometime, hard to describe

**Comment by mulshine** [comment]
words are hard

**Comment by spiricom** [comment]
The behavior may be as intended, but not what you expect:
We're currently imagining there as being a "working sequence", and then the stored sequences in the composition view. 
The randomize and clear buttons operate on the "working sequence". Is that what was confusing?

**Comment by JoshuaStorm** [comment]
Nah, that seems pretty natural.

Maybe this example makes more sense:

1. I have a handful of saved sequences, 1-n
2. I set working sequence to sequence 1
3. I hit full-randomize button, this seems to result in change pattern order which is fine.
4. I switch to another sequence, 2, I expect it to be exactly as I had saved sequence 2, but the changed pattern order persists.

This still might be intended, but I kind of like the ability to return completely to whatever saved state including the pattern order.

**Comment by spiricom** [comment]
yes, you are right, that's a bug.  Must be that we don't save pattern order as part of the sequence, or don't retrieve it properly.
